.github/workflows/ci.yml

```
@@ -0,0 +1,82 @@
+      - name: Lint with flake8
+        run: |
+          pipenv run flake8 .
+
+      - name: Test with pytest
+        # Ignores exit code 5 (no tests collected)
+        run: |
+          pipenv run pytest || [ $? -eq 5 ]
+
+      - name: Validate output with scrapy
+        if: github.event_name == 'pull_request'
+        run: |
```

actionlint (1.7.9)

shellcheck reported issue in this script: SC2046:warning:2:14: Quote this to prevent word splitting

(shellcheck)